### PR TITLE
Added messages hash, refactored main menu logic 

### PR DESCRIPTION
**Pull request by chrisstankus99**
Added a hash to store messages in and refactored multiple methods in game class. Also fixed the render method logic in the cell class. 

All tests pass, 100% coverage.

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -63,4 +63,12 @@ def consecutive_number_coordinates?(coordinates)
   def place(ship, coordinates)
     coordinates.each { |coordinate| @cells[coordinate].place_ship(ship) }
   end
+
```
Comment (tylertomlinson) [inline lib/board.rb:66]: Good work on making this render.

File: lib/game.rb [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+require './lib/cell'
+
+class Game
+
+  attr_reader :board, :cruiser, :submarine
+  def initialize
+    @board = Board.new
+    @cruiser = Ship.new("Cruiser", 3)
+    @submarine = Ship.new("Submarine", 2)
+    @messages = {
```
Comment (tylertomlinson) [inline lib/game.rb:12]: This looks awesome

File: test/game_test.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require_relative 'test_helper'
+require './lib/ship'
+require './lib/cell'
+require './lib/board'
+
+# class Gametest < Minitest::Test
```
Comment (tylertomlinson) [inline test/game_test.rb:6]: Are you testing anything within the game class?

File: lib/game.rb [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+  def start
+    puts @messages[:msg_3]
+    puts @messages[:msg_4]
+    puts @messages[:msg_5]
+    puts @messages[:msg_6]
+    input
+  end
+
+  def input
```
Comment (tylertomlinson) [inline lib/game.rb:56]: Somewhat hesitant to merge with master having a lot of commented out code. I think as long as we are covering 100% on Simplecov it won't be a problem.

File: lib/game.rb [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    until valid_coordinate?(shot_coordinate)
+      puts "Please enter a valid coordinate:\n"
+      shot_coordinate = gets.chomp.upcase
+    end
+    puts "Your shot on #{shot_coordinate} was a miss."
+    puts "My shot on #{computer_shot_coordinate} was a miss"
+    turn
+  end
+
+  def sanitized_input(user_input)
```
Comment (tylertomlinson) [inline lib/game.rb:98]: We can mock out user input for this once player and cpu are done.

File: lib/game.rb [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+  def menu_options(user_input)
+    if  user_input == "p"
+      start
+    elsif user_input == "q"
+      exit
+    else puts @messages[:err_msg_1]
+    end
+  end
+
+  def error_messages(coordinates)
```
Comment (tylertomlinson) [inline lib/game.rb]: Talked to Tim about this and it's better to use return. However we are nesting these within the hash so I think we can get rid of this method.

File: test/game_test.rb (hunk 3 above)
Comment (chrisstankus99) [reply test/game_test.rb:6]: No, not yet. I've been manually testing as I go. I'll work on testing the methods in here.

File: lib/game.rb (hunk 4 above)
Comment (chrisstankus99) [reply lib/game.rb:56]: Simplecov is showing 100% coverage.. I have removed some of the unneeded comments to clean this up a bit.

File: lib/game.rb (hunk 6 above)
Comment (chrisstankus99) [reply lib/game.rb]: Removed it! I forgot to earlier.

File: test/game_test.rb (hunk 3 above)
Comment (tylertomlinson) [reply test/game_test.rb:6]: ok cool. It might be a good idea to test through running and then using Simple to test what isnt covered.

**Pull request merged**